docs/reference/monitoring/indices.asciidoc

```diff
@@ -1,79 +1,143 @@
 [role="xpack"]
 [[config-monitoring-indices]]
-== Configuring indices for monitoring
+== Configuring data streams/indices for monitoring
 
-Sometimes the default index settings might not work for your situation. For
-example, you might want to change index lifecycle management (ILM) settings,
-add custom mappings, or set the number of shards and replicas
-for the monitoring indices.
+Monitoring data is stored in data streams or indices in {es}. The default data
+stream or index settings may not work for your situation. For example, you might
+want to change index lifecycle management (ILM) settings, mappings, or the
```

Comment:
I would not recommend at all to modify the mappings. Only index settings and ILM policies.

Reply:
I'd leave it the way it was phrased before, _"add custom mappings"_, since that can be legitimate use case and isn't potentially destructive like a modification